test/sshkit_functional_test.exs

```diff
@@ -5,116 +5,118 @@ defmodule SSHKitFunctionalTest do
 
   @bootconf [user: "me", password: "pass"]
 
-  @tag boot: [@bootconf]
-  test "connects", %{hosts: [host]} do
-    [{:ok, output, 0}] =
-      host
-      |> SSHKit.context()
-      |> SSHKit.run("id -un")
-
-    name = String.trim(stdout(output))
-    assert name == host.options[:user]
-  end
-
-  @tag boot: [@bootconf]
-  test "runs commands", %{hosts: [host]} do
-    context = SSHKit.context(host)
+  describe "run/2" do
```

Comment:
All that changed in this commit is that the following tests are grouped into this `describe` block.